### ISSUE TITLE
security/acme-client: add support for dnsapi "Euserv.eu"

### ISSUE THREAD

**Issue by prunkster** [opnsense/plugins#1779]
Hello,

I'm currently migrating to the ACME plugin for certificate management with DNS validation. Unfortunately, Euserv.eu seems to be missing under the DNS services (supported by ACME.sh: https://github.com/acmesh-official/acme.sh/wiki/dnsapi#47-use-euserveu-api).

It would be great if support for Euserv.eu api could be added to this plugin.

Thanks!

**Comment by fraenki** [comment]
@prunkster Would you mind to submit a [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)? It's not too hard, look at [this](https://github.com/opnsense/plugins/pull/1567/files) old PR for an example. :)

**Comment by fraenki** [comment]
This will be available in the upcoming os-acme-client release 1.31.

**Comment by prunkster** [comment]
Hi @fraenki,
sorry, I have been busy after work trying to debug some problems with my own implementation of this one. I just got it working last night...

While looking at the current master, most of the changes made are identical. There are two additional changes which I think are needed to get it working:
- As stated in the acme.sh documentation, Euserv is using a self-signed certificate for their API endpoint. Therefore I added the option "--insecure" to the acme.sh call (done via $acme_hook_options, hope that's the right way to go)
- I increased the maximum DNS sleep time to 84600 seconds (one day), as DNS records created via the API seemed to take quite long to be applied. While a record created via the web interface takes up to 30 minutes max (after my tests), the records created via API could not be resolved even after more than three hours (but are shown in the web interface). I don't know how long it took exactly, at least the next day they were resolvable too.

The last point was a bummer to me, wish I would have realized it earlier...

I created a pull request (based on the current master branch) with the aforementioned changes.

Thanks!
